File: sentimental-mario-less/mario.py
n = -2
while n <= 0 or n >= 9:
    try:
        n = int(input('Height: '))
    except:
        n = -1
i = n
while i != 0:
    j = i - 1
    k = i - 1
    while j != 0:
        print(' ', end="")
        j = j - 1
    while k != n:
        print('#', end="")
        k = k + 1
    i = i - 1
    print("")

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    int i, j;

    do
    {
        h = get_int("Height: ");
    }
    while (h <= 0 || h > 8);

    //loop the lines
    for (i = 1; i <= h; i++)
    {
        //loop the chars in the line and put space
        for (j = 1; j < h - i + 1; j++)
        {
            putchar(' ');
        }
        //loop the chars in the line and put # in the end
        // put the number of # equal to the line number
        for (j = h - i + 1; j <= h; j++)
        {
            putchar('#');
        }
        // put the space
        putchar(' ');
        putchar(' ');
        // put the number of # equal to the line number
        for (j = h - i + 1; j <= h; j++)
        {
            putchar('#');
        }
        putchar('\n');
    }
}